doc/reference/configuration/configuration_reference.rst

```diff
@@ -3133,6 +3133,35 @@ log.syslog.*
     | Default: box.NULL
     | Environment variable: TT_LOG_SYSLOG_SERVER
 
+.. _configuration_reference_lua:
+
+lua
+---
+
+The ``lua`` section outlines the configuration parameters related to the ``lua`` environment within Tarantool.
+
+Below is an overview of the available configuration options for ``lua``:
```

Comment:
We keep introductory texts in big references to the minimum, so this sentence looks redundant. This is also consistent with other sections of this page.